pkg/controller/machinehealthcheck/machinehealthcheck_controller_test.go

```diff
@@ -327,3 +327,32 @@ func TestUnhealthyForTooLong(t *testing.T) {
 		}
 	}
 }
+
+func TestApplyRemediationReboot(t *testing.T) {
```

Comment:
can you just set the type of remediation on the MHC object and run reconcile?

Reply:
that'd be a different unit test, we can add the case to testReconcile though

Reply:
done